ui/server_item.py

```diff
@@ -191,6 +198,9 @@ def apply_result(self):
             main_company.list_job(self)
 
     def exception_risen(self, exception):
+        """
+        Handle the exception if anything goes wrong.
```

Comment:
A note on style. I think "Handle the exception ..." is more correct, rather than "Handles the exception..." style that is used other docstrings.

Reply:
Seeing it now, I think "It handles ..." fits better because the docstring is the definition of the function itself and not a comment on it.

Reply:
Is that the recommended way for qgis plugins? I'm following Django style here (https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/)

Reply:
Ok. Let's keep it as "Handle". Will have to change other comments, though.